comments_blackbox_test.go

```diff
@@ -0,0 +1,190 @@
+package main_test
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/almighty/almighty-core"
+	"github.com/almighty/almighty-core/account"
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/app/test"
+	"github.com/almighty/almighty-core/gormapplication"
+	"github.com/almighty/almighty-core/gormsupport"
+	"github.com/almighty/almighty-core/gormsupport/cleaner"
+	"github.com/almighty/almighty-core/resource"
+	testsupport "github.com/almighty/almighty-core/test"
+	almtoken "github.com/almighty/almighty-core/token"
+	"github.com/almighty/almighty-core/workitem"
```

Comment:
It'd be nice to add a line in between almighty imports and external imports from other repos that don't belong to the go library.